src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,52 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
+
+  static deadAnimal(animal) {
```

Comment:
Please, fix the method name

```suggestion
  static removeEaten(animal) {
```

```suggestion
  static removeDeadAnimal(animal) {
```